wagtail_transfer/serializers.py

```diff
@@ -9,6 +11,65 @@
 from .models import get_base_model
 
 
+def _get_subclasses_recurse(model):
+    """
+    Given a Model class, find all related objects, exploring children
+    recursively, returning a `list` of strings representing the
+    relations for select_related, adapted from https://github.com/jazzband/django-model-utils/blob/master/model_utils/managers.py
+    """
+
+    related_objects = [f for f in model._meta.get_fields() if isinstance(f, models.OneToOneRel)]
+
+    rels = [
+        rel for rel in related_objects
+        if isinstance(rel.field, models.OneToOneField)
+        and issubclass(rel.field.model, model)
+        and model is not rel.field.model
+        and rel.parent_link
+    ]
+
+    subclasses = []
+    for rel in rels:
+        for subclass in _get_subclasses_recurse(rel.field.model):
+            subclasses.append(
+                rel.get_accessor_name() + LOOKUP_SEP + subclass)
+        subclasses.append(rel.get_accessor_name())
+    return subclasses
+
+
+def _get_sub_obj_recurse(obj, s):
+    """
+    Given an object and its potential subclasses in lookup string form,
+    retrieve its most specific subclass recursively
+    Taken from: https://github.com/jazzband/django-model-utils/blob/master/model_utils/managers.py
+    """
```

Comment:
I'm sure the recursion here could be avoided just by splitting the string on LOOKUP_SEP and looping over the resulting list, but I guess it's not worth the extra overhead of writing new code :-)